spec/rspec/core/option_parser_spec.rb

```diff
@@ -317,6 +340,24 @@ def generate_help_text
       end
     end
 
+    describe "--bisect" do
+      it "sets the `:bisect` option" do
+        options = Parser.parse(%w[ --bisect ])
+
+        expect(options[:bisect]).to be_truthy
+      end
+
+      it "sets the `:runner` option with the `bisect` invocation" do
+        bisect = double(:bisect)
+        allow(RSpec::Core::Invocations).to receive(:bisect).and_return(bisect)
```

Comment:
Stubbing invocations to return a double just so we can assert on equality below seems pretty sub-optimal.  As I see it, the `RSpec::Core::Invocations` methods are essentially just exposing constants -- given that they take no arguments.  So WDYT about turning them into constants, and then you don't have to stub here -- instead, down below it can do:

``` ruby
expect(options[:runner]).to eq(RSpec::Core::Invocations::BISECT)
```

Thoughts?